homeassistant/components/rest/switch.py

```diff
@@ -97,8 +98,9 @@ async def async_setup_platform(
             "Missing resource or schema in configuration. "
             "Add http:// or https:// to your URL"
         )
-    except (asyncio.TimeoutError, aiohttp.ClientError):
+    except (asyncio.TimeoutError, aiohttp.ClientError) as exc:
         _LOGGER.error("No route to resource/endpoint: %s", resource)
```

Comment:
Please put the log message as an exception argument for `PlatformNotReady` instead. Then we'll make sure not to spam the log on every retry.

Reply:
Hi, added in 55e4c2b  I think that solves this suggestion ?